marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -370,7 +387,7 @@ function addEntitySpecificProperties(result, entityInfo, selectedPropertyMetadat
     console.log(`Unable to obtain document with URI '${result.uri}'; will not add document metadata to its search result`);
   }
   result.entityInstance = entityInstance;
-  result.sources = getEntityInstanceHeaders(result.uri) ? getEntityInstanceHeaders(result.uri) : result.sources;
+  result.sources = getEntitySources(result.uri) ? getEntitySources(result.uri) : result.sources;
```

Comment:
You can remove the "?" operator here because getEntitySources always returns an array, right?